content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+ 
+### Tokens de Contas de serviço
+ 
+Uma conta de serviço é um autenticador habilitado automaticamente que usa bearer tokens para verificar as requisições. O plugin aceita dois parâmetros opcionais:
+ 
+* `--service-account-key-file` Um arquivo contendo uma chave codificada no formato PEM para assinar _bearer tokens_. Se não especificado, a chave privada de TLS servidor de API será utilizada
+* `--service-account-lookup` Se habilitado, _tokens_ deletados do servidor de API serão revogados.
+ 
+Contas de serviço são normalmente criadas automaticamente pelo servidor de API e associada a _pods_ rodando no cluster através do controlador de admissão [Admission Controller](/docs/reference/access-authn-authz/admission-controllers/). `ServiceAccount`. Contas podem ser explicitamente associadas com _pods_ utilizando o campo `serviceAccountName` na especificação do pod (`PodSpec`):
+ 
+{{< note >}}
+`serviceAccountName` é normalmente omitida por ser feito automaticamente
+{{< /note >}}
+ 
+```yaml
+apiVersion: apps/v1 # esta apiVersion é relevante a partir do Kubernetes 1.9
+kind: Deployment
+metadata:
+ name: nginx-deployment
+ namespace: default
+spec:
+ replicas: 3
+ template:
+   metadata:
+   # ...
+   spec:
+     serviceAccountName: bob-the-bot
+     containers:
+     - name: nginx
+       image: nginx:1.14.2
+```
+ 
+Tokens de Contas de serviço são perfeitamente válidos para ser usados fora do cluster e podem ser utilizados para criar identidades para processos de longa duração que desejem comunicar-se com a API do Kubernetes. Para criar manualmente uma conta de serviço, utilize-se simplesmente o comando `kubectl create serviceaccount (NOME)`. Isso cria uma conta de serviço e um segredo associado a ela no namespace atual.
+ 
+```bash
+kubectl create serviceaccount jenkins
+```
+ 
+```none
+serviceaccount "jenkins" created
+```
+ 
+Verificando um segredo associado:
+ 
+```bash
+kubectl get serviceaccounts jenkins -o yaml
+```
+ 
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+ # ...
+secrets:
+- name: jenkins-token-1yvwg
+```
+O segredo criado irá armazenar a autoridade de certificado do servidor de API e um JSON Web Token (JWT) digitalmente assinado.
+ 
+```bash
+kubectl get secret jenkins-token-1yvwg -o yaml
+```
+ 
+```yaml
+apiVersion: v1
+data:
+ ca.crt: (APISERVER'S CA BASE64 ENCODED)
+ namespace: ZGVmYXVsdA==
+ token: (BEARER TOKEN BASE64 ENCODED)
+kind: Secret
+metadata:
+ # ...
+type: kubernetes.io/service-account-token
+```
+ 
+{{< note >}}
+Valores são codificados em base64 porque segredos são sempre codificados neste formato.
+{{< /note >}}
+ 
+O JWT assinado pode ser usado como um _bearer token_ para autenticar-se como a conta de serviço. Veja [acima](#Colocando um _bearer token_  em uma requisição) para como o _token_ pode ser incluído em uma requisição. Normalmente esses segredos são montados no pod para um acesso interno ao cluster ao servidor de API, porém pode ser utilizado fora do cluster também.
```

Comment:
Here it's a reference to a section, doesn't need to be translated, just referenced to the section name in pt:
`[acima](#Colocando um _bearer token_  em uma requisição)`

Reply:
Resolved at 9df194e